server/proxy/proxy.go

```diff
@@ -42,6 +43,17 @@ func (p *Proxy) Call(sessionID, actingUserID string, creq *apps.CallRequest) *ap
 	if err != nil {
 		return apps.NewProxyCallResponse(apps.NewErrorCallResponse(err), metadata)
 	}
+
+	if creq.Path == "" || creq.Path[0] != '/' {
```

Comment:
maybe this block can be factored out into some utility method re: `.../server/http/gateway/static.go::cleanStaticPath()`

Reply:
The code is a bit different and `cleanStaticPath` is used for - well - well path of static assets. I'm fine with adding a  `cleanPath` method to keep things DRY, but I'm hesitant to mix up file paths and call paths.

Reply:
I was referring to the code snippet of `cleanStaticPath` that was doing the same as here, not to the whole method.

Reply:
Refactored the code in dd0244e161b4713bfef6650b496e7162713e5d63